src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -388,7 +391,8 @@
             :cipher-suites           (get-cipher-suites-config config)
             :protocols               (get-ssl-protocols-config config)
             :client-auth             (get-client-auth! config)
-            :ssl-crl-path            (get-ssl-crl-path! config)})))
+            :ssl-crl-path            (get-ssl-crl-path! config)
+            :disallow-renegotiation  (or (:disallow-renegotiation config) default-disallow-renegotiation)})))
```

Comment:
I assume the end result of a user's actions would cause `(disallow-renegotiation config)` to return `false` if they wanted to change this setting? If they did, wouldn't this still evaluate to `true` because of the default? How does a user enable renegotiation?

Reply:
Good point and thanks for catching it. Got left out when I switched to using disallow instead of allow semantics midway to avoid using an 'else' in Clojure that I initially struggled with :-)
In any case it is possible to over ride the default of disallowing renegotiations by adding "allow-renegotiation" to true in appropriate configuration files (like pe-puppet-server.conf). 
Do I need to document it someplace say in the main TK-webserver-jetty9 readme ?

Reply:
> Do I need to document it someplace

Yes. Webserver config docs live [here](https://github.com/puppetlabs/trapperkeeper-webserver-jetty9/blob/master/doc/jetty-config.md)